### DISCUSSION TITLE
What variables can be used in "Custom Script Template"?

### DISCUSSION THREAD

**Discussion by Rikk** [violentmonkey/violentmonkey#1961] [Q&A] [answered]
I know some are used by default, eg. {{date}} {{name}} {{url}}
But are there others? I would like to customize the format of date.

**Comment by tophf** [comment]
There are no other variables. How do you want to customize it?

**Reply by Rikk** [reply]
yyyy.mm.dd is the format I use for the version of my scripts.

**Reply by tophf** [reply]
I guess we can add `{{date:FMT}}` where FMT is MomentJS-compatible format used by our "Export to zip" file name option.

**Comment by tophf** [comment, marked as answer]
Implemented as `{{date:format}}` in 3c050449.
